build/sage_bootstrap/package.py

```diff
@@ -2,7 +2,7 @@
 """
 Sage Packages
 """
-
+from __future__ import annotations
```

Comment:
Unfortunately sage_bootstrap is not the right package for working with static typing.
We use this package to bootstrap the build of the project even on ancient platforms that provide only Python 2.7 or Python 3.6.

Reply:
Hence, please remove type annotations here and in the other script